From cc895ebc019a7d8e1bb4f0e0951dab83b4915d21 Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Fri, 10 Apr 2020 13:37:42 -0700
Subject: [PATCH 13/19] Clarify $imap_fetch_chunk_size is referring to headers.

!81 was opened because the documentation was confusing.  Hopefully
this rewording will make it clear the "size" refers to number of
headers per request.
---
 init.h | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/init.h b/init.h
index 1621f8af..6cbc30a3 100644
--- a/init.h
+++ b/init.h
@@ -1520,10 +1520,11 @@ struct option_t MuttVars[] = {
   { "imap_fetch_chunk_size",	DT_LNUM, R_NONE, {.p=&ImapFetchChunkSize}, {.l=0} },
   /*
   ** .pp
-  ** When set to a value greater than 0, new headers will be downloaded
-  ** in sets of this size.  If you have a very large mailbox, this might
-  ** prevent a timeout and disconnect when opening the mailbox, by sending
-  ** a FETCH per set of this size instead of a single FETCH for all new
+  ** When set to a value greater than 0, new headers will be
+  ** downloaded in groups of this many headers per request.  If you
+  ** have a very large mailbox, this might prevent a timeout and
+  ** disconnect when opening the mailbox, by sending a FETCH per set
+  ** of this many headers, instead of a single FETCH for all new
   ** headers.
   */
   { "imap_headers",	DT_STR, R_INDEX, {.p=&ImapHeaders}, {.p=0} },
